fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/BatchServiceImpl.java

```diff
@@ -60,8 +61,9 @@
      * be either persisted or written to a distributed map or sth, not just this
      * plain hashmap that follows
      */
-    private static Map<String, FedoraSession> transactions = new ConcurrentHashMap<>();
+    private static Map<String, FedoraSession> sessions = new ConcurrentHashMap<>();
 
+    @VisibleForTesting
```

Comment:
This is not actually used in any testing code... and can presently be safely `private`.